OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -431,9 +435,21 @@ void RemoveShellmapUI()
 			rootMenu.Parent.RemoveChild(rootMenu);
 		}
 
+		public static string PickLastModifiedMap(ModData modData)
```

Comment:
The naming/logic here is weird, because if the last modified map didn't change between calls of this method we return `null` instead of the last modified map. 🤔

Reply:
that's the point, perhaps the name is indeed misleading

Reply:
added a comment